doc/source/async_api.rst

```diff
@@ -0,0 +1,213 @@
+The Async Ray API
+=================
+
+Many web applications & prediction serving service today heavily rely on asynchronous execution to gain performance.
+Since version 3.5, a set of basic async interfaces has been introduced to Python,
+making it more feasible to program asynchronously,
+and popular async frameworks (aiohttp, aioredis, etc.) are emerging,
+indicating the power of asynchronous APIs.
+
+This document talks about asynchronous APIs in Ray.
+
+Starting Ray
+------------
+
+Please refer to `Starting Ray`_ for instructions.
+
+.. _`Starting Ray`: http://ray.readthedocs.io/en/latest/tutorial.html#starting-ray
+
+
+Initialize & shutdown Async Ray APIs
+------------------------------------
+
+Async Ray APIs can be initialized automatically by calling any related functions.
+But users could call `ray.experimental.async_api.init` ahead of time.
```

Comment:
Should we just remove the init() call then if it's optional? Seems like it can be confusing.